### PR TITLE
explain why the angles do not match

### LINKED ISSUE #66: Resolve th angle mismatch for (113): mode? constraints?
@ambarb : Found the FIXME item in `tst_e4cv_fourc.ipynb`:

```
(bluesky_2021_1) prjemian@poof ~/.../Bluesky/hklpy $ git grep FIXME | grep examples/
examples/archive/e6c_sixc.ipynb:    "# FIXME:  must get same numbers, probably in different rows\n",
examples/archive/hkl-example.ipynb:    "# FIXME: So far, only works when previous constraints are removed.  Wrong mode?\n",
examples/archive/hkl-example.ipynb:    "# FIXME: pick mode\n",
examples/archive/tardis_example.ipynb:    "# FIXME: hack to get around what should have been done at init of tardis_calc instance\n",
examples/archive/tardis_example.ipynb:    "    # energy=tardis_calc.energy, # FIXME: unexpected keyword argument\n",
examples/tst_e4cv_fourc.ipynb:    "# FIXME: (113) `th` values do not match."
```

_Originally posted by @prjemian in https://github.com/bluesky/hklpy/issues/52#issuecomment-748208500_

### PR DESCRIPTION
**Pull request by prjemian**
FIX #66

@ambarb Agree that this explanation is satisfactory? Paraphrasing:

> ... the UB matrix can't just be copied [from SPEC to hklpy] and re-used. Need to compute [the UB matrix] from the supplied reflection angles (which is why the orientation reflection data must also be written into the run, per #50).

### CODE REVIEW

File: examples/tst_e4cv_fourc.ipynb [hunk 1]
```diff
@@ -408,20 +417,11 @@
...
-      "E4CV [UB] [[-9.55498634e-02 -1.65427875e+00  2.42844498e-03] \n",
-      "           [ 2.63111155e-04  9.81585901e-03  1.65396189e+00] \n",
-      "           [-1.65871254e+00  9.82002627e-02 -3.89705597e-04]]\n",
-      "========= ===================================================\n",
-      "\n"
+     "output_type": "error",
+     "ename": "SyntaxError",
+     "evalue": "invalid syntax (<ipython-input-11-3719be6e20be>, line 18)",
+     "traceback": [
+      "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-11-3719be6e20be>\"\u001b[0;36m, line \u001b[0;32m18\u001b[0m\n\u001b[0;31m    tbl = pyRestTable.Table()compute\u001b[0m\n\u001b[0m                             ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
```
Comment (mrakitin) [inline examples/tst_e4cv_fourc.ipynb:424]: Why is there a syntax error?

File: examples/tst_e4cv_fourc.ipynb [hunk 2]
```diff
@@ -443,7 +443,7 @@
     "\n",
     "fourc.calc.sample.compute_UB(*refs)\n",
     "\n",
-    "tbl = pyRestTable.Table()\n",
+    "tbl = pyRestTable.Table()compute \n",
```
Comment (mrakitin) [inline examples/tst_e4cv_fourc.ipynb:446]: Should it be
```suggestion
    "tbl = pyRestTable.Table().compute \n",
```
?

**Review by mrakitin — CHANGES_REQUESTED** [review]
It seems there is an issue with the notebook (see the syntax error in https://nbviewer.jupyter.org/github/bluesky/hklpy/blob/602638df90dd729cd50fcaa7a2af8d020474dc41/examples/tst_e4cv_fourc.ipynb#Setup-the-UB-orientation-matrix-using-hklpy). Needs to be fixed before merging.

**Review by mrakitin — APPROVED** [review]

**Comment by prjemian** [comment]
@mrakitin Thanks!

**Pull request merged**